ios/browser/api/brave_shields/adblock_engine.h

```diff
@@ -80,30 +80,13 @@ OBJC_EXPORT
 - (NSString*)cosmeticResourcesForURL:(NSString*)url
     NS_SWIFT_NAME(cosmeticResourcesForURL(_:));
 
-/// Returns a stylesheet containing all generic cosmetic rules that begin with
```

Comment:
Unrelated line but `DomainResolverCallback` at the top of this file can be removed